src/test/kotlin/autolotto/LottoTest.kt

```diff
@@ -0,0 +1,122 @@
+package autolotto
+
+import autolotto.vo.AutoLotto
+import autolotto.vo.Lotto
+import autolotto.vo.WinningLotto
+import autolotto.winningpoint.WinningRank
+import io.kotest.matchers.ints.beInRange
+import io.kotest.matchers.shouldBe
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LottoTest {
+    @Test
+    fun `로또 1장은 6개의 숫자다`() {
+        // Given
+        val lotto = Lotto()
+
+        // When
+        val numbers = lotto.numbers
+
+        // Then
+        numbers.size shouldBe 6
+    }
+
+    @Test
+    fun `로또 1장의 숫자는 1~45의 숫자다`() {
+        // Given
+        val lotto = Lotto()
+
+        // When
+        val numbers = lotto.numbers
+
+        // Then
+        numbers.forEach {
+            it shouldBe beInRange(IntRange(1, 45))
+        }
+    }
+
+    @Test
+    fun `로또 1장의 숫자는 중복되지 않는다`() {
+        // Given
+        val lotto = Lotto()
+
+        // When
+        val numbers = lotto.numbers
+
+        // Then
+        numbers.distinct().size shouldBe 6
```

Comment:
```suggestion
        numbers.distinct() shouldHaveSize 6
```

p3; 사이즈도 메서드가 있습니다. 😊

Reply:
이게있을까 싶으면 요게 있군요 코테스트는!! 🐥👏🏻👏🏻👏🏻